marin/execution/executor.py

```diff
@@ -65,8 +65,15 @@
 
 - If we decide to rename fields, we can extend `versioned` to take a string of
   the old field name to preserve backward compatibility.
+
+- "Pseudo-dependencies" are dependencies that are not actually run, but are
```

Comment:
"not actually run"?  They have to be created somehow via a run, just not finished?